packages/components/CHANGELOG-FIGMA-COMPONENTS.md

```diff
@@ -1,4 +1,23 @@
 # [HDS Product - Components](https://www.figma.com/file/noyY6dUMDYjmySpHcMjhkN/HDS-Product---Components?type=design&node-id=6790-10926&mode=design&t=Ps0aMGZ6F3z7bAJ4-0)
+## December 20th, 2024
+
+`IconTile` - Visual updates made to the component, including removing the border and updating colors for improved contrast and alignment.
+
+`Figma Library 1.0` - **Deprecated**. It will no longer receive updates or support.
```

Comment:
I'm not 100% sure of the best way to format this, and all of the other release/deprecation notices, but I'll make a suggestion in here that might be appropriate based some of our other comms. Couple of notes:

- You could probably include the link in the backticks, or remove the backtick formatting altogether.
- It seems a bit repetitive to me (though open to other opinions) to have a separate line for each deprecation and release, I think you could probably combine these into a single line.

```suggestion
`[Figma Library v1.0](https://www.figma.com/design/noyY6dUMDYjmySpHcMjhkN/HDS-Components?m=auto&node-id=2-7&t=CPG3tlUvMHeQdEos-1)` - **Deprecated**. It will no longer receive updates or support. Instead, use the `[Figma Library v2.0](https://www.figma.com/design/iweq3r2Pi8xiJfD9e6lOhF/HDS-Components-v2.0?m=auto&node-id=2-7&t=9BNZJMDOrABsgaJQ-1)`
```

Reply:
I think the backticks should be removed since it's a file name and not a component name. Usually, we use backticks for components, styles, token names, etc.

I don't think we need the links though. If we want folks to move away from v1.0 I wouldn't direct them to the file. And the link to v2.0 should already be under the section heading, so that feels repetitive.